src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -4,215 +4,254 @@
 			<div class="tw-mb-2 md:tw-mb-0">
 				<kv-button
 					class="loan-filters__toggle rounded smallest secondary tw-mr-2"
+					variant="secondary"
 					@click.native.prevent="showFilters()"
 				>
 					Filter loans
 
-					<kv-icon
-						class="loan-filters__toggle-icon"
-						name="small-chevron"
-						:from-sprite="true"
-						aria-hidden="true"
+					<kv-material-icon
+						class=" tw-w-3"
+						name="chevron-down"
+						:icon="mdiChevronDown"
 					/>
 				</kv-button>
 				<span class="tw-font-medium tw-whitespace-nowrap">{{ totalCount }} loans</span>
 			</div>
 
 			<div v-if="showLoanDisplayToggle" class="loan-filters__loan-display">
-				<kv-pill-toggle
-					id="pill"
-					:options="[
-						{
-							title: 'Rows',
-							key: 'rows',
-						},
-						{
-							title: 'Grid',
-							key: 'grid',
-						},
-					]"
-					:selected="activeLoanDisplay"
-					@pill-toggled="(val) => { $emit('set-loan-display', val === 'rows') }"
-				/>
-			</div>
-		</div>
-
-		<div
-			v-if="filterChips.length"
-			class="tw-mt-3 tw-mx-2 tw-mb-1 md:tw-mx-7"
-		>
-			<div class="row">
 				<div
-					class="chips__container small-12 large-8 xxlarge-9 columns"
-					:class="{'chips--collapsed' : isChipsCollapsed}"
-					ref="chipsContainer"
+					class="tw-flex tw-cursor-pointer tw-items-center"
+					id="view-toggle"
 				>
-					<div
-						ref="chipsInnerContainer"
+					<span
+						class="tw-flex tw-items-center tw-flex-wrap"
+						:selected="activeLoanDisplay"
+						@click=" (val) => { $emit('set-loan-display', val === true) }"
```

Comment:
There is no longer a `val` to evaluate as that was emitted from KvPillToggle specifically.

You should be able to just use `@click="$emit('set-loan-display', true)"` here and `@click="$emit('set-loan-display', false)"` on the Grid span below